misc/docs/source/_templates/script.rst

```diff
@@ -1,10 +1,27 @@
+
+{%- set command_mapping = {
+    'obspy.scripts.flinnengdahl': 'obspy-flinn-engdahl',
+    'obspy.scripts.runtests': 'obspy-runtests',
```

Comment:
Aren't this one and a few others covered by the fallback for the `get`?

Reply:
I was thinking the second part of `command_mapping.get(fullname, 'obspy-' + objname.split('.')[-1])` should cover these ones and there's no need to list them. But actually, I'm not sure the difference between `fullname` and `objname`.

Reply:
oh right, the last part becomes the command. But for clarity I found it easier to add the full mapping, even if not really needed

Reply:
correct, better without fallback!